mozilla_django_oidc/auth.py

```diff
@@ -72,29 +72,60 @@ def create_user(self, claims):
 
         return self.UserModel.objects.create_user(username, email)
 
+    def get_algorithms(self):
+        """Returns the JWS verify algorithms"""
+        return ['HS256']
+
+    def get_verify_key(self):
+        """Returns the JWS verify key"""
+        return self.OIDC_RP_CLIENT_SECRET
```

Comment:
Why these two methods? I like that it at least makes it possible to change the algorithm (even though I don't know why you'd want to do that). 

The doc strings aren't particularly useful. They just say what the code already does. If the `'HS256'` value was moved out to a class attribute it'd be possible to override this class and override that value if need me. Same for `self.OIDC_RP_CIENT_SECRET`. I don't think that needs to be a method. 

Too many methods makes the code-reader think. "Why would I want to get the verifiy key??" If the point of the methods is to make inheritance easier then use the doc string to explain that. Or just leave them as a class attributes.